### ISSUE TITLE
Pages appearing on top of one another after mail merge

### ISSUE THREAD

**Issue by mattreub** [berteh/ScribusGenerator#84]
I designed the document on Scribus (a personalised diary to track specific things) and put the dates (which I wanted to use to populate the document) into excel and saved as a csv. Before merging the dates in, the document looked like week page trial.pdf and the file with the dates in was DATES TRIAL.xslx (obviously as a csv). However, when I merged the two, the pages in the final document appeared on top of one another, rather than being filled in the way I had envisaged, see week page mail merge.pdf

I'm not sure if this is my error or a bug, let alone how to fix it!

[Week page mail merge.pdf](https://github.com/berteh/ScribusGenerator/files/1011559/Week.page.mail.merge.pdf)
[DATES TRIAL.xlsx](https://github.com/berteh/ScribusGenerator/files/1011574/DATES.TRIAL.xlsx)
[Week page trial.pdf](https://github.com/berteh/ScribusGenerator/files/1011573/Week.page.trial.pdf)

**Comment by berteh** [comment]
Hello @mattreub 

The error you experience is "normal", you made no error. The reason is a former design choice to be able to support documents with pages of different sizes, explained in more details in#46.

But as the recent Scribus 1.5 does not support the display of more than 2 pages in a row anymore, this "feature" now becomes a nuisance, for your case of for others... so I'm looking into the best way to change it, and handle nicely documents with single page display as well as dual page display.

As long as you stick to single page display the fix proposed in #83 should work for you, and will be integrated in the next release of ScribusGenerator.

As long as you do not display pages in double

**Comment by jvr14115** [comment]
Do far no sollution?
I suppose it means, I need to somehow manage 1st and 2nd page in seperate lines and work with layers for 1st and 2nd page. Would work I guess.

**Comment by berteh** [comment]
No, so far no change here, as I find no "right" way to support templates with more pages. Any ideas?

**Comment by novicesmml** [comment]
I had the same issue as mattreub, and applied the fix recommended in #83. Now the pages come out separate but shifted up (not in the right place).
It is a document with 3 pages and in single page view, there is only one record that needs changing on the first page.  Besides this, the pdf file always comes out corrupt.
The expected output is one pdf file where the first every three pages are different.

Thank you very much for the effort you put into making this!
